Cargo.toml

```diff
@@ -3,6 +3,9 @@ members = [
   "crates/stdsimd-verify",
   "crates/stdsimd",
 ]
+exclude = [
+  "crates/wasm-test"
+]
```

Comment:
I think this can probably be removed now

Reply:
done